src/opnsense/mvc/app/views/OPNsense/Interface/vxlan.volt

```diff
@@ -0,0 +1,64 @@
+<script>
+    $( document ).ready(function() {
+        $("#grid-addresses").UIBootgrid(
+            {   search:'/api/interfaces/vxlan_settings/searchItem/',
+                get:'/api/interfaces/vxlan_settings/getItem/',
+                set:'/api/interfaces/vxlan_settings/setItem/',
+                add:'/api/interfaces/vxlan_settings/addItem/',
+                del:'/api/interfaces/vxlan_settings/delItem/'
+            }
+        );
+
+        $("#reconfigureAct").click(function(){
```

Comment:
I think I should componentize that since it is repeating over and over.

Reply:
I think that's the case for more frontend components at the moment, it would be good to look at that some point in time.